### PR TITLE
Adding primary_action and secondary_action tracks to Congrats Screen

### PR DESCRIPTION
**Pull request by ALEJOVILLA-ML**
## What have changed?
Adding primary_action and secondary_action tracks to Congrats Screen

### CODE REVIEW

File: ExampleSwift/ExampleSwift.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -298,13 +298,15 @@
...
 				"${BUILT_PRODUCTS_DIR}/MLUI/MLUI.framework",
+				"${BUILT_PRODUCTS_DIR}/PureLayout/PureLayout.framework",
 			);
 			name = "[CP] Embed Pods Frameworks";
 			outputPaths = (
 				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/AndesUI.framework",
 				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/FXBlurView.framework",
 				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/JRSwizzle.framework",
 				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/MLUI.framework",
+				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/PureLayout.framework",
```
Comment (Esteban-ML) [inline ExampleSwift/ExampleSwift.xcodeproj/project.pbxproj:309]: @jlainog  This PR is adding the PureLayout.framework removed in the PR I made before, I don't really understand why the bundle exec pod install behaves this way...

Should we keep it, right?

**Review by Esteban-ML — COMMENTED** [review]
LG! 🦦

**Review by Esteban-ML — APPROVED** [review]

File: ExampleSwift/ExampleSwift.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jlainog-meli) [reply ExampleSwift/ExampleSwift.xcodeproj/project.pbxproj:309]: we should try removing it explicitly just to make sure we are using it... if not we remove it completely.

File: MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXPaymentCongratsViewModel.swift [hunk 2]
```diff
@@ -272,10 +280,20 @@ extension PXPaymentCongratsViewModel: PXNewResultViewModelInterface {
...
+        paymentCongrats.mainAction?.action = action
         return paymentCongrats.mainAction
     }
 
     func getFooterSecondaryAction() -> PXAction? {
+        let action: (() -> Void) = { [unowned self] in
+            MPXTracker.sharedInstance.trackEvent(event: self.getSecondaryButtonTrack())
+            self.paymentCongrats.secondaryAction?.action()
+        }
+        paymentCongrats.mainAction?.action = action
```
Comment (jlainog-meli) [inline MercadoPagoSDK/MercadoPagoSDK/UI/PXResult/PXPaymentCongratsViewModel.swift:296]: looks like you are configuring the same `mainAction.action` for both main and secondary action...

**Review by jlainog-meli — CHANGES_REQUESTED** [review]
LG but I think there is an error when intercepting the actions

File: MercadoPagoSDK/MercadoPagoSDK/Core/PXAction.swift [hunk 3]
```diff
@@ -18,3 +18,12 @@ open class PXAction: NSObject {
         self.action = action
     }
 }
+
+extension PXAction {
+    func addTrackingEvent(_ trackingEvent: PXResultTrackingEvents) -> PXAction {
+        PXAction(label: self.label, action: {
```
Comment (jlainog-meli) [inline MercadoPagoSDK/MercadoPagoSDK/Core/PXAction.swift:24]: cool, but remember to check the linter that was raising a warning here

**Review by jlainog-meli — APPROVED** [review]

**Review by Esteban-ML — APPROVED** [review]

**Pull request merged**